content/en/blog/2023/2023-10-26-blixt-ebpf-rust-in-kubernetes-sigs.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: blog
+title: Blixt - A load-balancer written in Rust, using eBPF, born from Gateway API
+date: 2023-10-26
```

Comment:
This date is in the past. How about a date during KubeCon? Say, the 8th of November.

Reply:
Sounds good, updated in 57352fc94e323c34ddf0c59977b474b8f0be9796